src/pybind/mgr/ceph_module.pyi

```diff
@@ -2,6 +2,8 @@
 # Used by mypy to do proper type checking of mgr modules.
 # Without this file, all classes have undefined base classes.
 
+from typing import Any, Dict, List, Optional, Union
```

Comment:
Adding argument types to `ceph_module.pyi` only provides moderate gain,  as all of those methods are wrapped in mgr_module already.

Reply:
yeah, i knew.